src/index.js

```diff
@@ -0,0 +1,288 @@
+  'ctrl', 'win', 'alt', 'space', 'alt', '◀', '▼', '▶', 'ctrl',
+];
+
+const keyLayoutSh = [
+  '`', '!', '@', '#', '$', '%', '^', '&', '*', '(', ')', '_', '+', 'backspace',
+  'tab', 'Й', 'Ц', 'У', 'К', 'Е', 'Н', 'Г', 'Ш', 'Щ', 'З', 'Х', 'Ъ', '\u002F', 'del',
+  'caps', 'Ф', 'Ы', 'В', 'А', 'П', 'Р', 'О', 'Л', 'Д', 'Ж', 'Э', 'enter',
+  'shift', 'Я', 'Ч', 'С', 'М', 'И', 'Т', 'Ь', 'Б', 'Ю', '/', '▲', 'shift',
+  'ctrl', 'win', 'alt', 'space', 'alt', '◀', '▼', '▶', 'ctrl',
+];
+
+const english = [
+  '`', '1', '2', '3', '4', '5', '6', '7', '8', '9', '0', '-', '=', 'backspace',
+  'tab', 'q', 'w', 'e', 'r', 't', 'y', 'u', 'i', 'o', 'p', '[', ']', '|', 'del',
+  'caps', 'a', 's', 'd', 'f', 'g', 'h', 'j', 'k', 'l', ';', '"', 'enter',
+  'shift', 'z', 'x', 'c', 'v', 'b', 'n', 'm', ',', '.', '/', '▲', 'shift',
+  'ctrl', 'win', 'alt', 'space', 'alt', '◀', '▼', '▶', 'ctrl',
+];
+
+const Keyboard = {
+  elements: {
+    inp: null,
+    main: null,
+    keysContainer: null,
+    keys: [],
+    flag: false,
+  },
+
+  eventHandlers: {
+    oninput: null,
+    onclose: null,
+  },
+
+  properties: {
+    value: '',
+    capsLock: false,
+    lang: false,
+  },
+
+  events: {
+    availableLanguages: [russian, english],
+    languageNow: russian,
+  },
+
+  init() {
+    this.elements.inp = document.createElement('textarea');
+    this.elements.main = document.createElement('div');
+    this.elements.keysContainer = document.createElement('div');
+    this.elements.inp.classList.add('result');
+    this.elements.main.classList.add('keyboard');
+    this.elements.keysContainer.classList.add('keyboard--keys');
+    const now = localStorage.getItem('language_Now') || 0;
+    this.elements.keysContainer.appendChild(this.createKeys(this.events.availableLanguages[now]));
+    this.elements.flag = false;
+    document.addEventListener('keyup', (e) => {
+      if (e.code === 'ControlLeft') this.elements.flag = true;
+      if (e.code === 'AltLeft' && this.elements.flag) {
+        this.elements.flag = false;
+        this.changeLang();
+        this.elements.keysContainer.appendChild(this.createKeys(this.events.languageNow));
+        window.location.reload();
+      }
+    });
+    document.addEventListener('keyup', (e) => {
+      if (e.code === 'ControlRight') this.elements.flag = true;
+      if (e.code === 'AltRight' && this.elements.flag) {
+        this.elements.flag = false;
+        this.changeLang();
+        this.elements.keysContainer.appendChild(this.createKeys(this.events.languageNow));
+        window.location.reload();
+      }
+    });
+    this.elements.keys = this.elements.keysContainer.querySelectorAll('.keyboard--key');
+    this.elements.main.appendChild(this.elements.keysContainer);
+    document.body.appendChild(this.elements.inp);
+    document.body.appendChild(this.elements.main);
+
+    document.addEventListener('keydown', (e) => {
+      document.querySelector(`.keyboard--key[data="${e.code}` || e.which || +'"]').classList.add('active');
```

Comment:
please use only string literals without + ''

Reply:
as I can see `|| e.which` is never reached because `.keyboard--key[data="${e.code}` is always return true as non empty string